msteams-platform/sbs-gs-commandbot.yml

```diff
@@ -148,7 +148,7 @@ items:
 
     1. Select **Create a new Teams app** option to create an app using Teams Toolkit.
 
-        :::image type="content" source="../msteams-platform/assets/images/sbs-command-bot/create-new-app.png" alt-text="Wizard starts for creating a new project":::
+        :::image type="content" source="../msteams-platform/assets/images/sbs-notification-bot/create-new-app.png" alt-text="Wizard starts for creating a new project":::
```

Comment:
The alt-text isn't clear.

Reply:
@ashishguptaiitb,

We're editing the alt text in another PR, which we'll share soon